bindings/nodejs/examples/account-manager.js

```diff
@@ -1,23 +1,12 @@
+require('dotenv').config();
 const { AccountManager } = require('@iota/wallet');
 
-const accountManagerOptions = {
-    storagePath: './alice-database',
-    clientOptions: {
-        nodes: [
-            {
-                url: 'https://api.firefly.h.chrysalis-devnet.iota.cafe:14265/',
-                auth: null,
-                disabled: false,
-            },
-        ],
-        localPow: true,
-    },
-    secretManager: {
-        Mnemonic:
-            'acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast',
-    },
-};
+async function unlockAndReturnManager() {
```

Comment:
Not to sound pedantic, but perhaps we could rename this to something simpler? Just `unlockManager` or something along those lines.

Reply:
getUnlockedManager()?